packages/access-om3-nuopc/package.py

```diff
@@ -56,10 +57,17 @@ class AccessOm3Nuopc(CMakePackage):
     depends_on("mpi")
     depends_on("netcdf-fortran@4.6.0:")
     depends_on("esmf@8.3.0:")
-    depends_on("fms@2021.03:")
-    depends_on("parallelio@2.5.10:")
     depends_on("fortranxml@4.1.2:")
 
+    for compiler in supported_compilers():
+        if compiler == 'intel':
+            depends_on("fms@2021.03: build_type==RelWithDebInfo precision=64 +large_file ~gfs_phys ~quad_precision ~openmp", when="~openmp")
```

Comment:
Sorry, I was probably not very clear. Only the parallelio `fflags` and `cflags` are Intel specific. All the rest should be kept even when the compiler is not Intel. That means the FMS stuff should be moved out of the `for` loop and `build_type==RelWithDebInfo` should be added to parallelio in line 69.

Reply:
Addressed in https://github.com/ACCESS-NRI/spack-packages/pull/103/commits/2e8d3ba1df752d9deed30859d7e04099b4c2abe5, thanks for the heads up